src/bindings/js/node/src/addon.cpp

```diff
@@ -10,10 +10,11 @@
 #include "model_wrap.hpp"
 #include "node_output.hpp"
 #include "openvino/openvino.hpp"
-#include "preprocess/pre_post_process_wrap.hpp"
-#include "resize_algorithm.hpp"
 #include "tensor.hpp"
 #include "partial_shape_wrap.hpp"
+#include "preprocess/preprocess_wrap.hpp"
+
+Napi::PropertyDescriptor element = Napi::PropertyDescriptor::Accessor<enumElementType>("element");
 
```

Comment:
I propose to create a function that will expose `ov::preprocess` as ov addon property 
```suggestion
Napi::Value preprocess_namespace(const Napi::CallbackInfo& info) {
    auto namespace_property = Napi::Object::New(info.Env());
    // Here we will register all methods defined in separate header files
    Napi::PropertyDescriptor resizeAlgorithm = 
        Napi::PropertyDescriptor::Accessor<enumResizeAlgorithm>("resizeAlgorithm");
    namespace_property.DefineProperty(resizeAlgorithm);
    PrePostProcessorWrap::Init(info.Env(), namespace_property);

    return namespace_property;
}
```